src/sage/combinat/multiset_partition_into_sets_ordered.py

```diff
@@ -251,8 +251,8 @@ def _repr_normal(self):
             string_parts = (str(sorted(k)) for k in self)
         else:
             string_parts = (str(sorted(k, key=str)) for k in self)
-        string_parts = ", ".join(string_parts).replace("[","{").replace("]","}")
-        return "[" + string_parts + "]"
+        string = ", ".join(string_parts).replace("[", "{").replace("]", "}")
```

Comment:
isn't it a bad idea to call a variable `string` ? it's also a type.

Reply:
well, this is not a reserved keyword
```
sage: type(string)
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[1], line 1
----> 1 type(string)

NameError: name 'string' is not defined
```